azure-iot-device/azure/iot/device/common/transport/mqtt/mqtt_transport.py

```diff
@@ -0,0 +1,787 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
```

Comment:
It looks like this was a merge mistake.  This file moved and changed significantly from this version.

Reply:
Good catch, I'll scrap it. I knew something like this would happen.

---
In reply to: [282742575](https://github.com/Azure/azure-iot-sdk-python-preview/pull/86#discussion_r282742575) [](ancestors = 282742575)